### PR TITLE
bpmn-moddle: Updated type definitions for v8.0

### PR DESCRIPTION
**Pull request by Droid0012**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/bpmn-io/bpmn-moddle/blob/master/lib/bpmn-moddle.js#L69>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/bpmn-moddle/index.d.ts [hunk 1]
```diff
@@ -1,31 +1,23 @@
-// Type definitions for bpmn-moddle 5.1
+// Type definitions for bpmn-moddle 8.0
 // Project: https://github.com/bpmn-io/bpmn-moddle
 // Definitions by: DefinitelyTyped <https://github.com/DefinitelyTyped>
+//                 Artur Avetisov <https://github.com/Droid0012>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 5.2
```
Comment (peterblazejewicz) [inline types/bpmn-moddle/index.d.ts:6]: What's the reason to using latest TS? (can this be done in older syntax?)

Comment (rbuckton) [reply types/bpmn-moddle/index.d.ts:6]: I'm not sure I see what syntax warrants changing the minimum supported TypeScript version, it doesn't seem necessary.

Comment (jakebailey) [reply types/bpmn-moddle/index.d.ts:6]: Yeah, definitely remove this line entirely.

File: types/bpmn-moddle/index.d.ts [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
-    type GatewayDirection =
-        | "Unspecified"
-        | "Converging"
-        | "Diverging"
-        | "Mixed";
-    type ItemKind = "Physical" | "Information";
-    type MultiInstanceBehaviour = "None" | "One" | "All" | "Complex";
-    type ProcessType = "None" | "Public" | "Private";
-    type RelationshipDirection = "None" | "Forward" | "Backward" | "Both";
+    type AdHocOrdering = 'Parallel' | 'Sequential';
```
Comment (jakebailey) [inline types/bpmn-moddle/index.d.ts:9]: Most of this PR is just formatting changes. That's making it very hard to review.

**Review by sandersn — CHANGES_REQUESTED** [review]
Requesting changes so the bot knows the status.
Based on comments:

- please remove the typescript version line entirely
- revert the formatting changes and put them in a separate PR.

**Pull request closed without merging**